infrastructure/modules/elasticsearch/elasticsearch.tf

```diff
@@ -20,6 +20,11 @@ resource "aws_elasticsearch_domain" "elasticsearch_service" {
     }
   }
 
+  domain_endpoint_options {
+    enforce_https       = true
+    tls_security_policy = "Policy-Min-TLS-1-2-2019-07"
```

Comment:
Does this need to be moved into a VAR or is it going to be the same all the time?